### ISSUE TITLE
Version check on first step

### ISSUE THREAD

**Issue by erkage** [directus/app#2764]
I've just started to try out directus and when first try the admin/install and click on the first "next" button i've got an 500 error with a fatal error in the logs: `Declaration of Symfony\Component\Translation\TranslatorInterface::setLocale($locale) must be compatible...`

Then i checked again the "Minimum version" for PHP and the one that is installed on my server and it was just 7.1.3. So, updated and it works like a charm but then i've see that on the second page of the installation checks the versions and requirements also PHP. 

So to sum up it fails to start before the checks can run (in case of php) which is not the best order imho 🙂.

**Comment by benhaynes** [comment]
To achieve better clarity/visibility, we are now tracking feature requests within the [Feature Request](https://github.com/directus/api/projects/40) project board.

**This issue being closed does not mean it's not being considered.**
